config/cp.php

```diff
@@ -73,6 +73,44 @@
     |
     */
 
-    'link_to_docs' => true,
+    'link_to_docs' => env('STATAMIC_LINK_TO_DOCS', true),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Support Link
+    |--------------------------------------------------------------------------
+    |
+    | Set the location of the support link in the "Useful Links" header
+    | dropdown. Use 'false' to remove it entirely.
+    |
+    */
+
+    'support_url' => env('STATAMIC_SUPPORT_URL', 'https://statamic.com/support'),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Login Theme
+    |--------------------------------------------------------------------------
+    |
+    | Optionally spice up the login screen and choose
+    | between "rad" or "business" modes.
+    |
+    */
+
+    'login_theme' => env('STATAMIC_LOGIN_THEME', 'rad'),
+
```

Comment:
Since it's being applied to pages aside from the login view, should we instead just call this `theme` and `STATAMIC_THEME`?